api/envoy/api/v2/route/route.proto

```diff
@@ -921,6 +921,10 @@ message HeaderMatcher {
     //   "-1somestring"
     envoy.type.Int64Range range_match = 6;
   }
+
+  // If specified, header match will be performed based on if the header value is *not* the value
+  // specified or if the header is not present.  Defaults to false.
```

Comment:
nit: 1 space after period (@alyssawilk @htuch did our format checker somehow stop working for the proto comments or did it never work)?

Reply:
SUFFIXES in tools/check_format doesn't currently do protos.  I can take a look